src/openapi/components.yaml

```
@@ -0,0 +1,80 @@
+components:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)